plugins/sqliteendorsements/main.go

```diff
@@ -39,7 +39,7 @@ func (e *SqliteEndorsementStore) Init(args common.EndorsementStoreParams) error
 	}
 
 	e.db = db
-	e.path = dbPath
+	e.path = dbPath.(string)
```

Comment:
if `dbPath` is not a string this will panic at runtime.  Use `v, ok := ...` to catch unexpected types.

Reply:
Will discuss!